config/express.js

```diff
@@ -103,7 +103,7 @@ module.exports = (app, config) => {
   app.use(constants.SITE_ROOT, (err, req, res, next) => {
     const statusCode = err.statusCode || 500;
 
-    log.error({ err }, statusCode);
+    log.error(err, 'Error');
```

Comment:
I've changed this from the variable `statusCode` to the constant string `'Error'` for the reason that in the logs the message being the status code has no value as it is contained within the body of `err`. Additionally, if `err.statusCode` was not populated it would have been set to `500` which might not be accurate. Sending the status of `500` back from the application when an error has occurred and there is no known status seems to eb the correct thing to do so I've left it as is.